src/meta/app_env_validator.cpp

```diff
@@ -161,6 +161,7 @@ void app_env_validator::register_all_validators()
         {replica_envs::ROCKSDB_USAGE_SCENARIO, nullptr},
         {replica_envs::ROCKSDB_CHECKPOINT_RESERVE_MIN_COUNT, nullptr},
         {replica_envs::ROCKSDB_CHECKPOINT_RESERVE_TIME_SECONDS, nullptr},
+        {replica_envs::ROCKSDB_BLOCK_CACHE_ENABLED, nullptr},
```

Comment:
could you add a check function for ROCKSDB_BLOCK_CACHE_ENABLED?

Reply:
> could you add a check function for ROCKSDB_BLOCK_CACHE_ENABLED?

OK，I have added it.